Control-flow.Rmd

```diff
@@ -257,8 +257,8 @@ i
 \indexc{break}
 There are two ways to terminate a for loop early:
 
-* `next` terminates the current iteration and advances to the next.
-* `break` exits the entire for loop.
+* `next` stops the current iteration and advances to the next one.
```

Comment:
`one` feels weird to me here

Reply:
How about: `next` stops the current iteration and moves on to the next iteration.

Reply:
"rebinding" -> "updating"?

Reply:
Even iteration feels redundant to me. Updating sounds good

Reply:
'next' exits the current loop.
'break' exits the entire for loop.

Reply:
next exits the current iteration
break exits the entire for loop
?